### ISSUE TITLE
Backspace no longer working at all on iOS

### ISSUE THREAD

**Issue by chrisdeeming** [froala/wysiwyg-editor#4545]
##### Expected behavior.
Backspace should work

##### Actual behavior.
Backspace does not work.

##### Steps to reproduce the problem.
Try as directed in the GIF on iOS in a JSFiddle (or in a UI that isn't quite so non-responsive).

##### Editor version.
4.0.15

##### OS.
iOS (All)

##### Browser.
iOS (All)

##### Recording.
![image](https://user-images.githubusercontent.com/2158514/200277409-60424ca7-f003-4391-b32a-a4de12f578fb.png)

**Comment by chrisdeeming** [comment]
![75561BEE-74AB-4CEF-9B30-D0CC4042F367](https://user-images.githubusercontent.com/2158514/200296225-354340ce-51e7-490b-ad0e-fd9213840933.gif)

**Comment by Zenexer** [comment]
This may be related to #4546.

**Comment by brucetp** [comment]
I have users with the same issue with the delete key on Mobile IOS/Safari.

**Comment by AndrewBarba** [comment]
Major issue for us as well. Any update on a fix?

**Comment by chrisdeeming** [comment]
It's a shame that Froala are still making releases that contain new, show-stopping functionality. The stability of the editor, particularly with mobile devices, has always been sketchy. The attention given to these serious issues is lacking and there clearly isn't enough testing being done to catch these issues before release. Bugs happen but an essentially device breaking bug like this not being given near immediate attention is particularly disagreeable but, unfortunately, what we have come to expect at this point.

If there was an obvious migration path away without all of the pain associated with that, we'd be gone by now.

**Comment by TeaBough** [comment]
I'm considering those 2 alternatives, tiptap or lexical. If someone know another worth considering alternative, I'm all ears  😅

**Comment by mtran-method** [comment]
We're experiencing the same issue here. This is very detrimental to our business.

**Comment by brucetp** [comment]
Can Froala Support provide a status update on this very large issue??

**Comment by brucetp** [comment]
Got a response from a ticket I submitted.   They hope to have a fix by late December.  In the meantime they suggest to move back to version 4.0.14.  I'm using vue version 4.0.13 and I still saw the problem.  UNTIL I realized the dependency was still using v4.0.16 of the editor...  once I realized that I modified my lock file and all was well.

**Comment by jamesdixon** [comment]
Same issue here.

@brucetp thanks for the update!

**Comment by jafaripur** [comment]
Same issue, backspace not work on iOS.

**Comment by leokang94** [comment]
I have same issue.. so I tried using trick the following.
(I used react-froala-wysiwyg, so the details can be different.)

```tsx
// I'll skip the details.

import ReactFroalaWYSIWYG from 'react-froala-wysiwyg';

interface EditorProps {
  value: string;
  onChange: (value: string) => void;
}
export default function Editor({ value, onChange }: EditorProps) {
  const editorRef = useRef<ReactFroalaWYSIWYG>(null);

  // ...
  const config = {
    // ...
    events: {
      keyup: (e: KeyboardEvent) => {
      }
    }
  }
}

  return <ReactFroalaWYSIWYG tag="textarea" ref={editorRef}/>
}
```

**Comment by essmahr** [comment]
We also have a suggested workaround, though it's not pretty...

Because of the severity of some unresolved bugs, we've resorted to manually patching Froala source code (via search and replace at build time) to address the issues that Froala has not yet resolved themselves. Echoing the same feelings as @chrisdeeming, it's frustrating.

Regarding this specific issue, I believe this was introduced by trying to patch https://github.com/froala/wysiwyg-editor/issues/4480 in the v4.10.15 release.

In Froala 4.0.16 source code, replacing 

```
["INPUT","BUTTON","TEXTAREA","DIV","BODY"]
```

with 

```
["INPUT","BUTTON","TEXTAREA"]
```

...reverts the fix to https://github.com/froala/wysiwyg-editor/issues/4480 and resolves the mobile backspace bug (there are two occurrences of this array in the source).

#4480 is also a bad bug, but seems to only be a problem if you're using `enter: ENTER_P` (which we are not). If you are, you may be out of luck!

**Comment by ledicana** [comment]
Has anyone tried to update Froala to v4.0.17? The newest version should've fix this issue, but I'm having problems with the whole `editor` object. The `editor.table`, `editor.image`, `.link` and other basic properties of the `editor` object seem to be completely gone in the newest version.

**Comment by chrisdeeming** [comment]
Fixed in 4.0.17 allegedly. Anyone able to confirm?

**Comment by jafaripur** [comment]
I checked and issue resolved.
